src/grt/grt-avhpi.adb

```diff
@@ -27,6 +27,19 @@ with Grt.Vstrings; use Grt.Vstrings;
 with Grt.Rtis_Utils; use Grt.Rtis_Utils;
 with Grt.To_Strings;
 
+-- Acronyms that I don't understand:
+   -- Rti
+   -- Rtis
+   -- Rtin
+
+
+-- Questoins:
+   -- Explanation for the Ctxt objects would be really helpful.
+   -- Why are they used, and what do they represent?
+   -- Does ghdl support a VHPI interface?
+   -- What's the best way to test this work.  Should we add
+   --   a cocotb dependency to the testing?
```

Comment:
I really prefer to have standalone tests.  That's easier to investigate and do not depends on cocotb evolution.